src/plugins/files/server/integration_tests/file_service.test.ts

```diff
@@ -43,16 +44,15 @@ describe('FileService', () => {
   let fileServiceFactory: FileServiceFactory;
   let security: ReturnType<typeof securityMock.createSetup>;
   let auditLogger: AuditLogger;
+  let fileKindsRegistry: ReturnType<typeof getFileKindsRegistry>;
 
   beforeAll(async () => {
-    const { startES } = createTestServers({ adjustTimeout: jest.setTimeout });
-    manageES = await startES();
-    kbnRoot = createRootWithCorePlugins();
-    await kbnRoot.preboot();
-    await kbnRoot.setup();
-    coreStart = await kbnRoot.start();
+    const { startES, startKibana } = createTestServers({ adjustTimeout: jest.setTimeout });
+    const testServers = await Promise.all([startES(), startKibana()]);
```

Comment:
try starting kibana and ES together, granted kibana remains waiting for ES, but as soon as ES is ready kibana completes shortly after.